lib/gooddata/lcm/types/scalar/integer.rb

```diff
@@ -13,8 +13,8 @@ class IntegerType < BaseType
         CATEGORY = :scalar
         INTERNAL_TYPE = Integer
 
-        def check(_value)
-          true
+        def check(value)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)